components/GameCard/GameCard.js

```diff
@@ -21,9 +21,9 @@ export default function GameCard({
             <BsPersonCircle /> {name}
           </p>
         </PlayerDiv>
-        <DeleteButtonContainer>
+        <DeleteButtonContainer onClick={() => deleteCard(id)}>
           <DeleteButton>
-            <BsX onClick={() => deleteCard(id)} />
+            <BsX />
           </DeleteButton>
         </DeleteButtonContainer>
```

Comment:
you should set it on the `DeleteButton`. You can't set it directly via
**components/GameCard/GameCard.js**
```javascript
<DeleteButton handleClick={() => deleteCard(id)}>
  <BsX />
</DeleteButton>
```

**components/DeleteButton.js**
```javascript
export default function DeleteButton({ children, handleClick }) {
  return <StyledButton onClick={() => handleClick}>{children}</StyledButton>;
}
```

(`handleClick` is just a prop. Name it whatever u want)